app/Http/Controllers/ImportController.php

```diff
@@ -133,7 +133,7 @@ public function url(Request $request)
 				continue;
 			}
 			// Import photo
-			if (!$this->photo($tmp_name, $request['albumID'])) {
+			if (!$this->photo($tmp_name, Configs::get_value('delete_imported', '0'), $request['albumID'])) {
```

Comment:
actually...
The parameters of `$this->photo` are :
```
	 * @param $path
	 * @param bool $delete_imported
	 * @param int  $albumID
```
I don't think that `Configs::get_value('delete_imported', '0')` returns a boolean...

Reply:
Duh! You would think I would know that, considering that I added that parameter, and pretty recently at that...